mev_inspect/classifiers/specs/curve.py

```diff
@@ -0,0 +1,440 @@
+    ),
+    ClassifierSpec(
+        abi_name="StableSwapIronBank",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0x2dded6Da1BF5DBdF597C45fcFaa3194e53EcfeAF"],
+    ),
+    ClassifierSpec(
+        abi_name="StableSwapLink",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0xf178c0b5bb7e7abf4e12a4838c7b7c5ba2c623c0"],
+    ),
+    ClassifierSpec(
+        abi_name="StableSwapPAX",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0x06364f10B501e868329afBc005b3492902d6C763"],
+    ),
+    ClassifierSpec(
+        abi_name="StableSwaprenBTC",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0x93054188d876f558f4a66B2EF1d97d16eDf0895B"],
+    ),
+    ClassifierSpec(
+        abi_name="StableSwaprETH",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0xF9440930043eb3997fc70e1339dBb11F341de7A8"],
+    ),
+    ClassifierSpec(
+        abi_name="StableSwapsAAVE",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0xEB16Ae0052ed37f479f7fe63849198Df1765a733"],
+    ),
+    ClassifierSpec(
+        abi_name="StableSwapsBTC",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0x7fC77b5c7614E1533320Ea6DDc2Eb61fa00A9714"],
+    ),
+    ClassifierSpec(
+        abi_name="StableSwapsETH",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0xc5424B857f758E906013F3555Dad202e4bdB4567"],
+    ),
+    ClassifierSpec(
+        abi_name="StableSwapstETH",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0xDC24316b9AE028F1497c275EB9192a3Ea0f67022"],
+    ),
+    ClassifierSpec(
+        abi_name="StableSwapsUSD",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0xA5407eAE9Ba41422680e2e00537571bcC53efBfD"],
+    ),
+    ClassifierSpec(
+        abi_name="StableSwapUSDT",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0x52EA46506B9CC5Ef470C5bf89f17Dc28bB35D85C"],
+    ),
+    ClassifierSpec(
+        abi_name="StableSwapY",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0x45F783CCE6B7FF23B2ab2D70e416cdb7D6055f51"],
+    ),
+    ClassifierSpec(
+        abi_name="StableSwapYv2",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0x8925D9d9B4569D737a48499DeF3f67BaA5a144b9"],
+    ),
+    ClassifierSpec(
+        abi_name="DepositBUSD",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0xb6c057591E073249F2D9D88Ba59a46CFC9B59EdB"],
+    ),
+    ClassifierSpec(
+        abi_name="DepositCompound",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0xeB21209ae4C2c9FF2a86ACA31E123764A3B6Bc06"],
+    ),
+    ClassifierSpec(
+        abi_name="DepositPAX",
+        protocol=Protocol.curve,
+        valid_contract_addresses=["0xA50cCc70b6a011CffDdf45057E39679379187287"],
+    ),
+    ClassifierSpec(
+        abi_name="DepositsUSD",
```

Comment:
same comment as before that these can be a single ABI

Might be worth thinking about how we can put more metadata like that into these things, but for the purpose of arbs at least we'll be able to know the asset from the transfers inside

Reply:
I think the DepositsUSD is a staked usd deposit which is different than the other USDN/K/P contracts.